app/views/keywords/show.html.slim

```diff
@@ -0,0 +1,47 @@
+div class="d-flex flex-column align-items-center mt-4"
+
+  h1.display-3 == keyword.keyword
+
+  - if keyword.status == 'completed'
```

Comment:
Same above, checking enum helper, can we use `keyword.completed?`

Reply:
Fixed in 62e18feb5b5643d1766aa6f3826a73042d7b56cf